nemo/collections/asr/modules/rnnt.py

```diff
@@ -368,6 +368,35 @@ def batch_concat_states(self, batch_states: List[List[torch.Tensor]]) -> List[to
 
         return state_list
 
+    def batch_subset_states(
```

Comment:
Add this function to the abstract base class - and preferably make it abstract to force all subclasses to implement it. Please provide concrete implementation for all subclasses if it is indeed added to the abstract base class.